map/src/components/map.tsx

```diff
@@ -366,6 +385,22 @@ class MapComponent extends React.Component<Props, {}> {
     }
   };
 
+  private centerToGeolocation = () => {
+    navigator.geolocation.getCurrentPosition(position => {
```

Comment:
one final comment, let's also add an error handler that prints the error to the console, and just uses an `alert()` to display a message like "Unable to get geolocation".

Reply:
I'm using chrome too and it's working :) Let's see the error prompt then we can think through. It's probably a permission error. If then; we'll need to implement something like: 
```javascript
navigator.permissions.query({ name: 'geolocation' })
  .then((result) => {
    if (result.state == 'granted') {
     ...
    } else if (result.state === 'prompt') {
      ...
    } else if (result.state === 'denied') {
      // user must enable it manually once denied
      alert('Ask for to give permission manually')
    }
 });
```

Reply:
I'm actually getting an unusual response:

```
Network location provider at 'https://www.googleapis.com/' : No response received.
```

In any case, working well now i think!